index.js

```diff
@@ -1,6 +1,6 @@
 const fs = require('fs').promises
 const { compileTemplate } = require('@vue/compiler-sfc')
-const { optimize: optimizeSvg } = require('svgo')
+const { optimize } = require("svgo");
```

Comment:
Could you run the linter: `npm run lint`

Reply:
Sure, I'll do it this weekend. Didn't notice because there is no pre-commit hook, my bad.